api-catalog-ui/frontend/cypress/integration/e2e/dashboard/wizard-dialog.test.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-undef */
+/* eslint-disable spaced-comment */
+/// <reference types="Cypress" />
+
+function login() {
+    cy.visit(`${Cypress.env('catalogHomePage')}/#/`);
+    cy.url().should('contain', '/login');
+
+    const username = Cypress.env('username');
+    const password = Cypress.env('password');
+
+    cy.get('button[type="submit"').as('submitButton');
+
+    cy.get('#username').type(username);
+    cy.get('input[name="password"]').type(password);
+
+    cy.get('@submitButton').click();
+}
+
+describe('>>> Wizard Dialog test', () => {
+
+    it('Dialog test', () => {
+        login();
+
+        cy.get('.header').should('exist');
+
+        // Test first enabler
```

Comment:
Maybe split this whole test in several tests, for instance by enabler. It would be better to for readability of test report